tests/sessions_tests/tests.py

```diff
@@ -462,6 +475,8 @@ def test_clearsessions_command(self):
         storage_path = self.backend._get_storage_path()
         file_prefix = settings.SESSION_COOKIE_NAME
 
+        settings.SESSION_COOKIE_AGE = 0
```

Comment:
This should like go in `@override_settings` rather than modifying the setting directly which might leak to other tests.